tp_timesheet/__main__.py

```diff
@@ -23,11 +25,16 @@ def run():
     if URL is None: 
         raise ValueError("URL is not set, please run `export TPURL='<URL OF TIMESHEET>'")
     if EMAIL is None:
-        raise ValueError("EMAIL is not set, please run `export TPMAIL='<TP EMAIL ADDR>'")
+        raise ValueError("EMAIL is not set, please run `export TPEMAIL='<TP EMAIL ADDR>'")
 
     args = parse_args()
-
-    # TODO: Calculate Dates
+    if args.start == "today":
```

Comment:
```suggestion
    if args.start.lower() == "today":
```

This will allow "Today" or "TODAY".